adafruit_azureiot/device_registration.py

```diff
@@ -112,7 +112,7 @@ def _connect_to_mqtt(self) -> None:
             " - device_registration :: connect :: created mqtt client. connecting.."
         )
         while not self._auth_response_received:
-            self._mqtt.loop()
+            self._mqtt.loop(2)
```

Comment:
These are added because `MiniMQTT` will throw when the loop timeout is less then the socket_timeout, which defaults to `1` [here](https://github.com/adafruit/Adafruit_CircuitPython_MiniMQTT/blob/47422868d2ea782c865198d98dc502e4fd721f63/adafruit_minimqtt/adafruit_minimqtt.py#L1031C27-L1031C42)